web/src/main/java/com/epam/esm/controller/TagController.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.dto.TagDto;
+import com.epam.esm.service.TagService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@Validated
```

Comment:
May I ask you to explain why this annotation is needed?

Reply:
I removed this annotation from class level. I forgot to remove it before when was trying to set up validation.
commit 27fb4d1